Submitted by: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 20050502
Description:
  Removes standard_exec_prefix_2 from library search path when cross-compiling
  ie: if cross_compile = 1 in the specs file, do not add
      /usr/lib/gcc/${TARGET}/${GCC_VER} to the library search path
 
--- gcc-3.4.3/gcc/gcc.c-orig	2004-09-09 01:16:11.000000000 +1000
+++ gcc-3.4.3/gcc/gcc.c	2005-05-02 21:32:34.000000000 +1000
@@ -3765,8 +3765,11 @@
 
   add_prefix (&startfile_prefixes, standard_exec_prefix, "BINUTILS",
 	      PREFIX_PRIORITY_LAST, 1, warn_std_ptr, 0);
-  add_prefix (&startfile_prefixes, standard_exec_prefix_2, "BINUTILS",
+  if (*cross_compile == '0')
+    {
+      add_prefix (&startfile_prefixes, standard_exec_prefix_2, "BINUTILS",
 	      PREFIX_PRIORITY_LAST, 1, warn_std_ptr, 0);
+    }
 
   tooldir_prefix = concat (tooldir_base_prefix, spec_machine,
 			   dir_separator_str, NULL);
